ui/src/components/StoryDrawer.tsx

```diff
@@ -0,0 +1,119 @@
+import * as React from "react";
+import Drawer from "@material-ui/core/Drawer";
+import {
+  DrawerAuthorText,
+  DrawerContentText,
+  DrawerCountryText,
+  DrawerRightText,
+  DrawerTitleText,
+} from "../styles/typography";
+import ArrowForwardIcon from "@material-ui/icons/ArrowForward";
+import Button from "@material-ui/core/Button";
+import Grid from "@material-ui/core/Grid";
+import styled from "styled-components";
+import ChinaFlag from "../assets/flags/China.png";
+import TempShoe from "../assets/images/temp.png";
+import { colors } from "../styles/colors";
+
+const StyledIconButton = styled(Button)`
+  position: absolute;
+  left: 0px;
+  top: 0px;
+  && {
+    background-color: ${colors.secondary};
+    color: ${colors.white};
+    min-width: 66.77px;
+    min-height: 60px;
+    width: 66.77px;
+    height: 60px;
+    border-radius: 0px;
+    &:hover {
+      background-color: ${colors.secondaryDark1};
+    }
+  }
+`;
+
+const StyledImage = styled.img`
+  border-radius: 0px;
+  width: 100%;
+`;
+
+const StyledRoot = styled.div`
+  width: 100vh;
+  padding-left: 30vh;
+  padding-right: 30vh;
+`;
+
+const StyledFlagImage = styled.img`
+  border-radius: 10px;
+  width: 100%;
```

Comment:
Should be constrained to 50px by 35px

Reply:
Used the grid size suggestion you had instead